apis/pkg/v1alpha1/interfaces.go

```diff
@@ -275,6 +285,36 @@ func (p *ProviderRevision) SetControllerReference(c runtimev1alpha1.Reference) {
 	p.Status.ControllerRef = c
 }
 
+// GetCrossplaneVersion of this ProviderRevision.
+func (p *ProviderRevision) GetCrossplaneVersion() string {
+	return p.Status.Crossplane
+}
+
+// SetCrossplaneVersion of this ProviderRevision.
+func (p *ProviderRevision) SetCrossplaneVersion(v string) {
+	p.Status.Crossplane = v
+}
+
+// GetDependencies of this ProviderRevision.
+func (p *ProviderRevision) GetDependencies() []Dependency {
+	return p.Status.DependsOn
+}
+
+// SetDependencies of this ProviderRevision.
+func (p *ProviderRevision) SetDependencies(d []Dependency) {
+	p.Status.DependsOn = d
+}
+
+// GetPermissionRequests of this ProviderRevision.
+func (p *ProviderRevision) GetPermissionRequests() []rbac.PolicyRule {
+	return p.Status.PermissionRequests
+}
+
+// SetPermissionRequests of this ProviderRevision.
+func (p *ProviderRevision) SetPermissionRequests(r []rbac.PolicyRule) {
+	p.Status.PermissionRequests = r
+}
+
```

Comment:
Note that permission requests still exist on the PackageRevision interface and types, despite being removed from meta `Provider` type for now. I opted not to remove them since the `ConfigurationRevision` has had them from the beginning, despite the `Configuration` meta package type not supporting comparable fields. Still feels like best pattern would just be having a single `Package` and `PackageRevision` type for installing and managing packages, but :man_shrugging: